### ISSUE TITLE
Format of /skillsheet is off

### ISSUE THREAD

**Issue by tylerking** [Jessy1237/DwarfCraft#1]
I leveled up a few skills and printed out my /skillsheet and it displays like this: https://imgur.com/a/iffK2

**Comment by Jessy1237** [comment]
The first line i can't control as the character limit of the line is reached due to your user name being so long, thats a minecraft limit so i can't fix that. However i will look into the other columns being out of placed for the levels of the skills.

**Comment by tylerking** [comment]
I understand that but its also a bit wordy. What about shortening it to something like:

SkillSheet - doitliketyler [Catfolk - Lvl 2]

**Comment by Jessy1237** [comment]
This issue should be fixed with the latest alteration commit.
